src/components/site/SiteNavButton.md

```diff
@@ -0,0 +1,3 @@
+```jsx
+<SiteNavButton />
```

Comment:
Only seeing a markdown file for the SiteNavButton. Will probably need a file for this if we want to test on the styleguide page as well.